File: Year 1/Semester 2/Java/Lessons/else_and_if.java
import java.util.Scanner;

public class else_and_if {
    public static void main(String[] args) {
        
        Scanner sca = new Scanner(System.in);
        String s = sca.nextLine();
       
        if (s.equals("tim")) {
                System.out.println("The name is tim");
        }
        else if (s.equals("hello")) {
            System.out.println("hello, You typed" + s);  
        }
        else {
            System.out.println("You typed" + s);
        }
    }
    
}

File: Year 1/Semester 1/C/Lessons/else_and_if.c
#include <stdio.h>
#include <string.h>

int main() {
    char s[100];  // Buffer to hold the input string

    // Taking input from the user
    printf("Enter a string: ");
    fgets(s, sizeof(s), stdin);  // Use fgets to read a line of input
    s[strcspn(s, "\n")] = '\0';   // Remove the newline character at the end if present

    // If-else statements for string comparison
    if (strcmp(s, "tim") == 0) {
        printf("The name is tim\n");
    } else if (strcmp(s, "hello") == 0) {
        printf("hello, You typed %s\n", s);
    } else {
        printf("You typed %s\n", s);
    }

    return 0;
}
